commit f1e0c51a67fb4fa009ed383d4f758e89c81b6a5f (refs/changes/95/2395/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-08-15T19:08:39+00:00 (2 years, 2 months ago)
    
    MANTA-3375 AMON alarm "upset.manta.postgres.long_query" hardcodes path to "psql"
    Reviewed by: Trent Mick <trentm@gmail.com>
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/alarm_metadata/probe_templates/postgres.yaml b/alarm_metadata/probe_templates/postgres.yaml
index 97b55e3..323e5f3 100644
--- a/alarm_metadata/probe_templates/postgres.yaml
+++ b/alarm_metadata/probe_templates/postgres.yaml
@@ -243,7 +243,7 @@
         -
             type: cmd
             config:
-                cmd: "/opt/local/bin/psql -U postgres -c \"SELECT pid,usename,client_addr,client_port,waiting,state,NOW() - query_start AS elapsed, query FROM pg_stat_activity WHERE state != 'idle' AND query NOT LIKE 'autovacuum: %' AND NOW() - query_start > interval '120 seconds'\" | tail +3"
+                cmd: "PATH=/opt/postgresql/current/bin:/opt/local/bin:$PATH psql -U postgres -c \"SELECT pid,usename,client_addr,client_port,waiting,state,NOW() - query_start AS elapsed, query FROM pg_stat_activity WHERE state != 'idle' AND query NOT LIKE 'autovacuum: %' AND NOW() - query_start > interval '120 seconds'\" | tail +3"
                 interval: 60
                 timeout: 30
                 threshold: 2
